packages/devextreme/testing/tests/DevExpress.ui.widgets/listParts/commonTests.js

```diff
@@ -4410,7 +4465,7 @@ QUnit.module('Accessibility', () => {
             allowItemDeleting: true,
         });
 
-        const $items = $list.find(toSelector(LIST_ITEM_CLASS));
```

Comment:
If you remove all using of `toSelector`, let's also remove this function declaration

Reply:
[e6e2060](https://github.com/DevExpress/DevExtreme/pull/27230/commits/e6e2060bea7bb96bcb81ebb10616ccb090464477)